.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
What are these files in .idea folder are for?

Reply:
@liornoy 
I usually use PyCharm and it automatically creates "idea" folder that contains settings for the project. I accidentally added it to the PR. I will remove it and add it to the git ignore so it won't happen again.